scss/_colors.scss

```diff
@@ -0,0 +1,6 @@
+@include oColorsSetColor('white-slate-20', oColorsMix($color: 'white', $background: 'slate', $percentage: 20));
+@include oColorsSetColor('slate-white-40', oColorsMix($color: 'slate', $background: 'white', $percentage: 40));
```

Comment:
Do any of these colours align with what you've added to `o-colors`?

Reply:
No, 'slate-white-15` and `slate-white-5` are the new "internal" shades of slate. I left existing slate/white mixes unchanged for now.